packages/notebook/style/index.css

```diff
@@ -274,32 +274,6 @@
   line-height: 1.4;
 }
 
-
-/*-----------------------------------------------------------------------------
-| Output Area View
-|----------------------------------------------------------------------------*/
-
-
-.jp-LinkedOutputView .jp-OutputArea-prompt {
-  display: none;
-}
-
-.jp-LinkedOutputView .jp-OutputArea {
-  height: calc(100% - var(--jp-toolbar-micro-height));
-  display: block;
-}
-
-.jp-LinkedOutputView .jp-OutputArea-child:only-child {
-  height: calc(100% - var(--jp-toolbar-micro-height));
-}
-
-.jp-LinkedOutputView-toolbar {
```

Comment:
This `box-shadow` styling that the notebook has and the linked output used to have are kind of nice. Maybe we should bring them over to the `MainAreaWidget`?

Reply:
Good idea.